reps/2023-02-03-node-affinity-feature-enhancements.md

```diff
@@ -0,0 +1,437 @@
+## Summary
+### General Motivation
+
+The current node affinity feature is relatively simple, it can only affinity with one node. 
+Many general functions cannot be supported.
+
+Current node affinity feature don't support functions:
+1. Don't support affinity to a batch of nodes. Current only support affinity to one node.
+2. Don't support anti-affinity to a node of a batch of nodes. 
+3. Don't support to classify nodes by labels, and then affinity or anti-affinity to a certain type of nodes.
+4. Don't support affinity/anti-affinity to nodes by node ip or node name. Current only support node id identification.
+
+This REP is to solve the above problems.
+
+
+### Should this change be within `ray` or outside?
+
+Yes, this will be a complement to ray core's ability to flexibly schedule actors/tasks/node.
+
+## Stewardship
+### Required Reviewers
+ 
+@ericl @stephanie-wang @wumuzi520 SenlinZhu @Chong Li  @scv119 (Chen Shen) @Sang Cho @jjyao (Jiajun Yao) @Yi Cheng
+### Shepherd of the Proposal (should be a senior committer)
+
```

Comment:
This is a required piece of information. I suggest @scv119

Reply:
Agree . I will add it.